heareval/predictions/task_predictions.py

```diff
@@ -742,13 +752,26 @@ def print_scores(mode, scores_and_trainers):
         scores_and_trainers.append((best_model_score, trainer, predictor))
         print_scores(mode, scores_and_trainers)
 
+    def vstr(v):
```

Comment:
vstr stands for value str? This is for serializing values to prepare for json? I know you want this short but the intentions of this function aren't super clear. `serialize_value(str: v)`?

Reply:
Done, good idea